### PR TITLE
Add content for README

### PR DESCRIPTION
**Pull request by PhilipGlover**
PSC-37

### CODE REVIEW

**Comment by eedwards0** [comment]
Please add a WEB_LOGGING_LEVEL configuration variable

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+psc_submissions| the filing resource including id, dates, PSC id and etag, and links | this and the database will be created by the service upon submitting filing details|
+
+Configuration
+-------------
+System properties for psc-filing-api are defined in `application.properties`. These are normally configured per environment.
+
+Variable| Description                                                                                 | Example  |
+--------------------|---------------------------------------------------------------------------------|------------|
+MONGO_PSC_API_DB_NAME| The name of the collection responsible for storing PSC filings                              | collection_name |
+MONGODB_URL| The URL of the MongoDB instance where documents and application data should be stored       | mongodb://mongohost:27017|
```
Comment (eedwards0) [inline README.md:44]: Please add a database name to the URL

Comment (PhilipGlover) [reply README.md]: added `transacations_pscs` to config examples

**Review by eedwards0 — APPROVED** [review]
Updates completed

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+REF_SYMBOL_SET| Set of characters permitted in randomly generated submission numbers          | abc123                                                                         |
+MANAGEMENT_ENDPOINTS_ENABLED_BY_DEFAULT|                             | false                                                                          |
+MANAGEMENT_ENDPOINT_HEALTH_ENABLED|                             | true                                                                           |
+MANAGEMENT_ENDPOINTS_WEB_PATH_MAPPING_HEALTH|                 | healthcheck                                                                    |
+MANAGEMENT_ENDPOINTS_WEB_BASE_PATH|                              | /persons-with-significant-control                                              |
+LOGGING_LEVEL| Log message granularity                         | INFO                                                                           |
+REQUEST_LOGGING_LEVEL| Request log message granularity                       | WARN                                                                           |
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply README.md]: added

**Pull request merged**